<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zoxel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/gfx/icons/block.png</normaloff>:/images/gfx/icons/block.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="glparent" native="true">
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_saveas"/>
    <addaction name="separator"/>
    <addaction name="action_export_troxel"/>
    <addaction name="action_export_image"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_axis_grids"/>
    <addaction name="action_wireframe"/>
    <addaction name="action_voxel_edges"/>
    <addaction name="separator"/>
    <addaction name="action_zoom_in"/>
    <addaction name="action_zoom_out"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
    <addaction name="separator"/>
    <addaction name="action_rotate_x"/>
    <addaction name="action_rotate_y"/>
    <addaction name="action_rotate_z"/>
    <addaction name="separator"/>
    <addaction name="action_mirror_x"/>
    <addaction name="action_mirror_y"/>
    <addaction name="action_mirror_z"/>
    <addaction name="separator"/>
    <addaction name="action_resize"/>
   </widget>
   <widget class="QMenu" name="menuAnimation">
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="title">
     <string>Animation</string>
    </property>
    <addaction name="action_anim_play"/>
    <addaction name="action_anim_stop"/>
    <addaction name="action_anim_next"/>
    <addaction name="action_anim_previous"/>
    <addaction name="separator"/>
    <addaction name="action_anim_add"/>
    <addaction name="action_anim_copy"/>
    <addaction name="separator"/>
    <addaction name="action_anim_delete"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuAnimation"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolbar">
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="separator"/>
   <addaction name="action_undo"/>
   <addaction name="action_redo"/>
   <addaction name="separator"/>
   <addaction name="action_axis_grids"/>
   <addaction name="action_wireframe"/>
   <addaction name="action_occlusion"/>
   <addaction name="separator"/>
   <addaction name="action_voxel_colour"/>
   <addaction name="action_background"/>
   <addaction name="action_resize"/>
   <addaction name="separator"/>
   <addaction name="action_reset_camera"/>
   <addaction name="action_zoom_in"/>
   <addaction name="action_zoom_out"/>
  </widget>
  <widget class="QToolBar" name="toolbar_drawing">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dock">
   <property name="windowTitle">
    <string>Palette</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="palette">
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>0</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLineEdit" name="paletteRGBvalue">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>#ff0000</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolbar_animation">
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_anim_add"/>
   <addaction name="action_anim_copy"/>
   <addaction name="action_anim_stop"/>
   <addaction name="action_anim_play"/>
   <addaction name="action_anim_previous"/>
   <addaction name="action_anim_next"/>
  </widget>
  <action name="action_exit">
   <property name="icon">
    <iconset>
     <normaloff>:/gfx/gfx/application-exit.png</normaloff>:/gfx/gfx/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset>
     <normaloff>:/gfx/gfx/help-contents.png</normaloff>:/gfx/gfx/help-contents.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/disk-return-black.png</normaloff>:/images/gfx/icons/disk-return-black.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_axis_grids">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/grid.png</normaloff>:/images/gfx/icons/grid.png</iconset>
   </property>
   <property name="text">
    <string>Axis Grids</string>
   </property>
   <property name="toolTip">
    <string>Toggle Axis Grids</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+1</string>
   </property>
  </action>
  <action name="action_wireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/border.png</normaloff>:/images/gfx/icons/border.png</iconset>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+2</string>
   </property>
  </action>
  <action name="action_new">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/document.png</normaloff>:/images/gfx/icons/document.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_saveas">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/disk-return-black.png</normaloff>:/images/gfx/icons/disk-return-black.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/folder-horizontal-open.png</normaloff>:/images/gfx/icons/folder-horizontal-open.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_background">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/palette.png</normaloff>:/images/gfx/icons/palette.png</iconset>
   </property>
   <property name="text">
    <string>Background</string>
   </property>
   <property name="toolTip">
    <string>Choose background colour</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+6</string>
   </property>
  </action>
  <action name="action_resize">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/transform-scale-2.png</normaloff>:/images/gfx/icons/transform-scale-2.png</iconset>
   </property>
   <property name="text">
    <string>Resize</string>
   </property>
   <property name="toolTip">
    <string>Set dimensions</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+7</string>
   </property>
  </action>
  <action name="action_occlusion">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/weather-clear-2.png</normaloff>:/images/gfx/icons/weather-clear-2.png</iconset>
   </property>
   <property name="text">
    <string>Occlusion</string>
   </property>
   <property name="toolTip">
    <string>Render with ambient occlusion</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+4</string>
   </property>
  </action>
  <action name="action_reset_camera">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/zoom-original-4.png</normaloff>:/images/gfx/icons/zoom-original-4.png</iconset>
   </property>
   <property name="text">
    <string>Reset Camera</string>
   </property>
   <property name="toolTip">
    <string>Reset camera position</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="action_voxel_edges">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/categories.png</normaloff>:/images/gfx/icons/categories.png</iconset>
   </property>
   <property name="text">
    <string>Voxel Edges</string>
   </property>
   <property name="toolTip">
    <string>Toggle view voxel edges</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+3</string>
   </property>
  </action>
  <action name="action_voxel_colour">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/color.png</normaloff>:/images/gfx/icons/color.png</iconset>
   </property>
   <property name="text">
    <string>Voxel Colour</string>
   </property>
   <property name="toolTip">
    <string>Pick Voxel Colour</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+5</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/undo.png</normaloff>:/images/gfx/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/redo.png</normaloff>:/images/gfx/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="action_zoom_in">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/zoom_in.png</normaloff>:/images/gfx/icons/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="action_zoom_out">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/zoom_out.png</normaloff>:/images/gfx/icons/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_anim_add">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/Create.png</normaloff>:/images/gfx/icons/Create.png</iconset>
   </property>
   <property name="text">
    <string>Add Frame</string>
   </property>
   <property name="toolTip">
    <string>Add new animation frame</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
   <action name="action_anim_copy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/redo.png</normaloff>:/images/gfx/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Copy Frame</string>
   </property>
   <property name="toolTip">
    <string>Copy animation frame</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_anim_play">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/Go.png</normaloff>:/images/gfx/icons/Go.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play animation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="action_anim_stop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/Stop.png</normaloff>:/images/gfx/icons/Stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop animation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="action_anim_next">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/Goforward.png</normaloff>:/images/gfx/icons/Goforward.png</iconset>
   </property>
   <property name="text">
    <string>Next Frame</string>
   </property>
   <property name="toolTip">
    <string>Go to next frame</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="action_anim_previous">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/Goback.png</normaloff>:/images/gfx/icons/Goback.png</iconset>
   </property>
   <property name="text">
    <string>Previous Frame</string>
   </property>
   <property name="toolTip">
    <string>Go to previous frame</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="action_anim_settings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/wrench-screwdriver.png</normaloff>:/images/gfx/icons/wrench-screwdriver.png</iconset>
   </property>
   <property name="text">
    <string>Animation Settings</string>
   </property>
   <property name="toolTip">
    <string>Animation Settings</string>
   </property>
  </action>
  <action name="action_anim_delete">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/Cancel.png</normaloff>:/images/gfx/icons/Cancel.png</iconset>
   </property>
   <property name="text">
    <string>Delete Frame</string>
   </property>
   <property name="toolTip">
    <string>Delete current frame</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_rotate_x">
   <property name="text">
    <string>Rotate Model X-Axis</string>
   </property>
   <property name="toolTip">
    <string>Rotate voxels around X axis</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_rotate_y">
   <property name="text">
    <string>Rotate Model Y-Axis</string>
   </property>
   <property name="toolTip">
    <string>Rotate voxels around Y axis</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_rotate_z">
   <property name="text">
    <string>Rotate Model Z-Axis</string>
   </property>
   <property name="toolTip">
    <string>Rotate voxels around Z axis</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_export_image">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/gfx/icons/disk.png</normaloff>:/images/gfx/icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>Export Image...</string>
   </property>
   <property name="toolTip">
    <string>Export image of current model view</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_mirror_x">
   <property name="text">
    <string>Mirror Model X-Axis</string>
   </property>
   <property name="toolTip">
    <string>Mirror voxels in X axis</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="action_mirror_y">
   <property name="text">
    <string>Mirror Model Y-Axis</string>
   </property>
   <property name="toolTip">
    <string>Mirror voxels in Y axis</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="action_mirror_z">
   <property name="text">
    <string>Mirror Model Z-Axis</string>
   </property>
   <property name="toolTip">
    <string>Mirror voxels in Z axis</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_export_troxel">
   <property name="text">
    <string>Export Troxel-link...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
